Code/GraphMol/MolDraw2D/MolDraw2DUtils.h

```diff
@@ -159,6 +159,44 @@ RDKIT_MOLDRAW2D_EXPORT inline void contourAndDrawGaussians(
   contourAndDrawGaussians(drawer, locs, heights, widths, nContours, levels, ps);
 };
 
+//! add R/S and E/Z annotation to atoms and bonds respectively.
+RDKIT_MOLDRAW2D_EXPORT inline void addStereoAnnotation(const ROMol &mol) {
+  for (auto atom : mol.atoms()) {
+    if (atom->hasProp("_CIPCode")) {
+      std::string lab = "(" + atom->getProp<std::string>("_CIPCode") + ")";
+      atom->setProp(common_properties::atomNote, lab);
+    }
+  }
+  for (auto bond : mol.bonds()) {
+    if (bond->getStereo() == Bond::STEREOE) {
+      bond->setProp(common_properties::bondNote, "(E)");
+    } else if (bond->getStereo() == Bond::STEREOZ) {
+      bond->setProp(common_properties::bondNote, "(Z)");
+    }
+  }
+};
+
+//! add R/S and E/Z annotation to atoms and bonds respectively.
```

Comment:
Change this comment?
It's also possible to call add[Atom,Bond]Indices multiple times and get 1,1,1,1,1 etc.  I guess users should be allowed to shoot their feet if they want too and it's probably not worth checking.

Reply:
Thanks for catching the screwed up comment.

It is a good point that these shouldn't really be part of the public API (since drawMolecule() calls them). I will fix that.